src/xfixes.rs

```diff
@@ -2,10 +2,104 @@
 // The X11 libraries are available under the MIT license.
 // These bindings are public domain.
 
-use super::xlib::XID;
+use libc::{c_char, c_int, c_short, c_uint, c_ulong, c_ushort};
+use xlib::{Atom, Bool, Cursor, Display, Pixmap, Status, Time, Window, XRectangle, GC, XID};
```

Comment:
It doesn't build correctly due to this use statement, a suggestion would be to use `super::`

Cheers

Reply:
It's been a while, I don't even remember why I dropped the `super::`.